packages/validation/src/validators/practices/rules/KBP107-noPodCreate.ts

```diff
@@ -4,6 +4,8 @@ import {isClusterRole} from '../../custom/schemas/clusterrole.rbac.authorization
 import {isRole} from '../../custom/schemas/role.rbac.authorization.k8s.io.v1.js';
 import {NSA_RELATIONS} from '../../../taxonomies/nsa.js';
 
+const KIND_CREATES_PODS = ['pods', 'deployments', 'statefulsets', 'jobs', 'cronjobs'];
```

Comment:
Looking at the [specs here](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#podtemplatespec-v1-core), it seems there are few more kinds which can create Pods:

![image](https://github.com/kubeshop/monokle-core/assets/1061942/0b33611d-ca33-46d7-a5fc-2dfa33c928e3)

AFAIU, all "Workloads APIs" can result in Pods creation:

![image](https://github.com/kubeshop/monokle-core/assets/1061942/9f4e1a74-cc6b-435d-8a2f-de33b58a7172)

Not 100% though, does it relates directly to this rule?

Reply:
Thanks I overlooked DaemonSet and ReplicaSet, ReplicationController is not really used afaik